integration-tests/src/test/resources/bash-scripts/appcheck.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
Can u re-name the file to CustomLivenessProbe.sh to provide the better context.

Reply:
Done